### DISCUSSION TITLE
New CUDA.jl release?

### DISCUSSION THREAD

**Discussion by GVigne** [JuliaGPU/CUDA.jl#1675] [General]
Hi!
I was wondering when the next release of the CUDA.jl package will take place. There have been implementations of wrappers for linear algebra functions (ex: 8b0c2fec3e0d5577020b7b7cfe0752863dc14bb8, 13d46d47f92268d6c449e2f62b181564b652d486...) since 3.12, and it would be nice to have an official CUDA version with them.
Are there any plans to do a new release? Is there anything I can do to help?

**Comment by maleadt** [comment]
Since the next release is going to breaking, I don't want to rush things. So thoroughly testing things would definitely be helpful! Some of the changes are pretty invasive (e.g. the revamped artifact loading, or the fact that CUDNN etc now live in their own submodule).

**Reply by GVigne** [reply]
Ok, I'm going to do some testing and I'll raise issues if I encounter a bug. Thanks!
